Source/WebCore/platform/graphics/PlatformColorSpace.h

```diff
@@ -46,9 +46,7 @@ class PlatformColorSpace {
 public:
     enum class Name : uint8_t {
         SRGB
-#if ENABLE(DESTINATION_COLOR_SPACE_LINEAR_SRGB)
```

Comment:
It sounds like a good idea to remove this, but not in this PR, it is way too convoluted now, with those unimportant changes mixed with actual code changes. How about we remove the option in a follow-up PR once this has landed?